wiki/Daily_challenge/en.md

```diff
@@ -0,0 +1,40 @@
+# Daily Challenge
+
+*Daily challenge* is a mode where the main task is to pass a selected beatmap that changes everyday sometimes with specific [mods](wiki/Gameplay/Game_modifier).
+
+## How to play
+
+*Notice: Daily challenge requires an [osu! account](/wiki/Registration), is not available for [silenced](/wiki/Silence) players and is only avaible in osu!(lazer).*
```

Comment:
```suggestion
*Notice: Daily challenge requires an [osu! account](/wiki/Registration), and is not available for [silenced](/wiki/Silence) players.*
```

Reply:
Maybe we should also say in the notice that dailies are only avaible for the osu! ruleset for now?
I feel it should be added since... well, it is only for osu! right now.

Reply:
Im not going to add this right away since i feel this needs a bit of a discussion since in the future it is planed to add dailies for every (main)  ruleset one by one

Reply:
If we are going to add this, then I would say in the first paragraph.

Something along the lines of `for the osu! game mode` might work.

Reply:
applied the said suggestion, will leave this open for now regardless.